src/pkg/runner/runner.go

```diff
@@ -52,6 +57,23 @@ func Run(tasksFile types.TasksFile, taskName string, setVariables map[string]str
 	return err
 }
 
+func (r *Runner) importTasks(includes []string) error {
+	// iterate through includes, open the file, and unmarshal it into a Task
+	for _, include := range includes {
+		taskFileDir := filepath.Dir(config.TaskFileLocation)
+		includePath := filepath.Join(taskFileDir, include)
+
+		var tasksFile types.TasksFile
+		if err := zarfUtils.ReadYaml(includePath, &tasksFile); err != nil {
+			return err
+		}
+
+		// add the tasks to the runner
+		r.TasksFile.Tasks = append(r.TasksFile.Tasks, tasksFile.Tasks...)
```

Comment:
Should the included task files also be sent to `importTasks`? The pattern should be fairly straightforward and it would provide a lot of flexibility to UDS CLI users

Reply:
doneskies